### DISCUSSION TITLE
Whats the point of X_PHPOS_FAKE?

### DISCUSSION THREAD

**Discussion by btzdnl** [rspamd/rspamd#4091] [Q&A]
I just received a mail that got hit by this rule. And it does hit hard by default. The header was:

    X-PHP-Originating-Script: 4062033:index.php

However, what _is_ spammy about 7-digit UIDs? Just because of some old-school conventions that UIDs should only go up to 65535? 4062033 is just a normal UID you could have on just about any reasonably modern Linux/Unix system.

This rules strikes me as super arbitrary.

**Comment by ntninja** [comment]
This was introduced in #1205. Maybe the original creator of that rules remembers is use?

**Reply by ntninja** [reply]
Opened PR: https://github.com/rspamd/rspamd/pull/5745
